docker/Dockerfile

```diff
@@ -164,6 +164,7 @@ RUN --mount=type=cache,target=/root/.cache/uv \
 # see https://github.com/pytorch/pytorch/pull/123243
 ARG torch_cuda_arch_list='7.0 7.5 8.0 8.9 9.0 10.0 12.0'
 ENV TORCH_CUDA_ARCH_LIST=${torch_cuda_arch_list}
+ENV TORCH_NVCC_FLAGS="-Xfatbin -compress-all -compress-mode=size"
```

Comment:
So this is TORCH"s NVCC flags, I think you may need to add it CMAKE_NVCC_FLAGS or something similar depending on your CMake infrastructure with NVCC (or just add it directly to the CMake).

Reply:
Yep I realized after looking into pytorch, added in https://github.com/vllm-project/vllm/pull/20694/commits/721dcb2ebc3c6322579bf463371fa9c53077c368